scripts/production/homology_stats_rapid.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This script fetches the number of coding genes and number of coding genes with homologies
+against the taxonomically closest reference species.
+
+Usage example::
+    python homology_stats_rapid.py\
```

Comment:
A blank line is needed here for the usage example to be recognised as a code block:
```suggestion
Usage example::

    python homology_stats_rapid.py\
```